apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/Defendant/Defendant.tsx

```diff
@@ -61,6 +67,8 @@ const Defendant = () => {
               nationalId: defendant.nationalId,
               noNationalId: defendant.noNationalId,
               citizenship: defendant.citizenship,
+            }).catch(() => {
```

Comment:
Consider removing this catch and handle the error in the useDefendant hook in a similar way as is done in the useCase hook.